bitbots_docs/docs/manual/hardware/bitfoot.rst

```diff
@@ -3,15 +3,52 @@ BitFoot
 =======
 
 The BitFoot is the foot pressure sensor by the Hamburg Bit-Bots.
-It features a higher update rate and theoretically higher precision than the `Rhoban ForceFoot <https://www.github.com/Rhoban/ForceFoot>`_ on which it is based.
+It features a higher update rate and higher resolution than the Rhoban ForceFoot_ on which it is based.
 
-Firmware, schematics and instructions how to flash firmware can be found in the git repository_:
+This board measures 4 differential voltage signals from load cells. It connects to a RS485 or TTL Bus compatible with Dynamixel motors from Robotis.
 
+We managed to achieve a sensor update rate of 697Hz. The board itself can be read faster than 1kHz from the Dynamixel bus.
+
+Firmware, schematics, gerber files, and BOMs can be found in our git repository_:
+
+In a previous version, we used an STM32F103 microcontroller on a BluePill board.
+Since we had some issues with the microcontrollers performance we switched to an ESP32.
+
+Since we did not want to redesign the analog part of the board, we designed an adapter board from the ESP32 to the BluePill pinout.
```

Comment:
```suggestion
Because we did not want to redesign the analog part of the board, we designed an adapter board from the ESP32 to the BluePill pinout.
```